scripts/lib/spreadsheet-standardizer.ts

```diff
@@ -79,17 +89,33 @@ export class SpreadsheetStandardizer {
     const output: Record<string, string> = {};
     for (const key in input) {
       const cleaned = cleanFieldName(key);
-      if (this.fieldMap[cleaned] === undefined && this.strict) {
+      if (this.fieldAliasToCanonical[cleaned] === undefined && this.strict) {
         throw new Error(`Invalid column found: ${cleaned}; original: ${key}`);
       }
 
-      const newCol = this.fieldMap[cleaned] || key;
+      const newCol = this.fieldAliasToCanonical[cleaned] || key;
       const val = this.convertToCanonical(input[key], newCol);
 
       output[newCol] = this.handleSpecialCases(val, newCol);
     }
     return output;
   }
+
+  convertToAliases(input: StringKeyed): StringKeyed {
+    const output: StringKeyed = {};
+    for (const [fieldName, field] of Object.entries(input)) {
+      let outputFieldName = fieldName;
+      if (this.fieldMap[fieldName] && this.fieldMap[fieldName][0]) {
```

Comment:
Wouldn't it be an error condition if this were false? It would mean there's a field in the JSON that isn't mapped to/from a spreadsheet column, and that shouldn't happen, right?

Reply:
See my response to your related comment – for collected data, we unfortunately do expect there may be columns that aren't part of our schema, and I think we should support that (Tom C told me as much).